packages/dita-example-sx-modules-spec-learning-xsd-learning-base-mod/src/configureSxModule.js

```diff
@@ -134,90 +135,90 @@ export default function configureSxModule(sxModule) {
 		blockHeaderLeft: [createMarkupLabelWidget()]
 	});
 
 	// lcPrereqs
 	//     The <lcPrereqs> describes the knowledge, experience, or other prerequisites needed to complete the
 	//     content.
-	configureAsFrame(sxModule, 'self::lcPrereqs', t('prerequisites'), {
+	configureAsFrame(sxModule, xq`self::lcPrereqs`, t('prerequisites'), {
 		contextualOperations: [
 			{ name: ':lcPrereqs-insert-title' },
 			{ name: ':contextual-delete-lcPrereqs' }
 		],
 		defaultTextContainer: 'p',
 		emptyElementPlaceholderText: t('type the prerequisites'),
-		titleQuery: './title',
+		titleQuery: xq`./title`,
 		blockHeaderLeft: [createMarkupLabelWidget()],
 		blockOutsideAfter: [createElementMenuButtonWidget()]
 	});
 
 	// lcResources
 	//     The <lcResources> provides a list of related resources and information about them, such as related
 	//     articles or samples on the web.
-	configureAsFrame(sxModule, 'self::lcResources', t('resources'), {
+	configureAsFrame(sxModule, xq`self::lcResources`, t('resources'), {
 		contextualOperations: [
 			{ name: ':lcResources-insert-title' },
 			{ name: ':contextual-delete-lcResources' }
 		],
 		defaultTextContainer: 'p',
 		emptyElementPlaceholderText: t('type the resources'),
-		titleQuery: './title',
+		titleQuery: xq`./title`,
 		blockHeaderLeft: [createMarkupLabelWidget()],
 		blockOutsideAfter: [createElementMenuButtonWidget()]
 	});
 
 	// lcReview
 	//     The <lcReview> provides a review of the main points.
-	configureAsFrame(sxModule, 'self::lcReview', t('review'), {
+	configureAsFrame(sxModule, xq`self::lcReview`, t('review'), {
 		contextualOperations: [
 			{ name: ':lcReview-insert-title' },
 			{ name: ':contextual-delete-lcReview' }
 		],
 		defaultTextContainer: 'p',
 		emptyElementPlaceholderText: t('type the review'),
-		titleQuery: './title',
+		titleQuery: xq`./title`,
 		blockHeaderLeft: [createMarkupLabelWidget()],
 		blockOutsideAfter: [createElementMenuButtonWidget()]
 	});
 
 	// lcSummary
 	//     The <lcSummary> provides a textual summary that describes the main learning goals and lessons
 	//     learned.
-	configureAsFrame(sxModule, 'self::lcSummary', t('summary'), {
+	configureAsFrame(sxModule, xq`self::lcSummary`, t('summary'), {
 		contextualOperations: [
 			{ name: ':lcSummary-insert-title' },
 			{ name: ':contextual-delete-lcSummary' }
 		],
 		defaultTextContainer: 'p',
 		emptyElementPlaceholderText: t('type the summary'),
-		titleQuery: './title',
+		titleQuery: xq`./title`,
 		blockHeaderLeft: [createMarkupLabelWidget()],
 		blockOutsideAfter: [createElementMenuButtonWidget()]
 	});
 
 	// lcTime
 	//     The <lcTime> specifies the time expected to complete an activity.
-	configureAsFrameWithBlock(sxModule, 'self::lcTime', t('time'), {
+	configureAsFrameWithBlock(sxModule, xq`self::lcTime`, t('time'), {
 		blockHeaderLeft: [createMarkupLabelWidget()],
 		blockHeaderRight: [
-			createLabelQueryWidget('./@value', {
+			createLabelQueryWidget(xq`./@value`, {
```

Comment:
Lets not upgrade this and leave a TODO instead

Reply:
Optionally, add something temporary locally to the LabelQueryWidget that warns when passing an XQExpression. That should let you find the other occurences as well